src/components/i18nComp.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <!-- <div>
+    <button @click="setLocale('en')">english</button>
+    <button @click="setLocale('bn')">Bengali</button>
+  </div> -->
+  <div class="lang-change">
+    <span :class="{ active: !localeValue }">EN</span
+    ><label class="switch">
+      <input v-model="localeValue" type="checkbox" @change="setLocale" />
+      <span class="slider"></span> </label
+    ><span :class="{ active: localeValue }">BN</span>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      localeValue: false,
+    };
+  },
+  methods: {
+    setLocale() {
+      if (this.localeValue) {
+        this.$i18n.locale = "bn";
+      } else {
+        this.$i18n.locale = "en";
+      }
+    },
+  },
+};
+</script>
+
+<style>
+.active {
+  color: rgb(255, 0, 131);
+}
+.lang-change {
+  display: flex;
+  min-width: 7rem;
+  justify-content: space-between;
+  align-items: center;
+}
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 50px;
```

Comment:
In one project, use only one type of unit. Somewhere rem is used so always use rem.

Reply:
only box shadows are left with px value